plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/DefaultThriftMetastoreClientFactory.java

```
@@ -129,12 +130,40 @@ private TTransport createTransport(HostAndPort address, Optional<String> delegat
         return Transport.create(address, sslContext, socksProxy, timeoutMillis, metastoreAuthentication, delegationToken);
     }
 
+    private static TrustManager[] getTrustAllCertsTrustManager()
+    {
+        TrustManager[] trustAllCerts = new TrustManager[] {
+                new X509TrustManager()
+                {
+                    @Override
+                    public java.security.cert.X509Certificate[] getAcceptedIssuers()
+                    {
+                        return new X509Certificate[0];
+                    }
+
+                    @Override
+                    public void checkClientTrusted(
+                            java.security.cert.X509Certificate[] certs, String authType)
+                    {
```

Should we throw an exception like we do in `OkhttpUtil#setupInsecureSsl`